src/components/navigators/BottomNavigator/BottomNavigator.style.ts

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+
+export const BottomNavContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  padding: 1rem;
+  background-color: ${(props: any) => props.theme.container_color};
```

Comment:
[ P3 ]

```suggestion
  background-color: ${({theme}) => theme.container_color};
```
요렇게 변경하실 수 있을것 같습니다!